packages/core/backend/sirius-components-collaborative/src/main/resources/schema/omnibox.graphqls

```diff
@@ -0,0 +1,25 @@
+extend type Viewer {
```

Comment:
two spaces instead of four for the indentation of graphqls files

Reply:
Done